Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
I believe that it was not included before because Godep was previously only vendoring two directories. You can see that change in Godeps.json.

I believe this, because I know that `go-autorest` test code was vendored, hence I know that the `-t` flag was used.

I'm fine with not vendoring the test code... but please note that we were already vendoring the `autorest` test code, so I would be undoing that as well, since it's a global knob, unless I'm misunderstanding a finer point of using `godep`.